packages/tupaia-web/src/features/Dashboard/ExportDashboard/Preview.tsx

```diff
@@ -0,0 +1,232 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { useParams } from 'react-router';
+import downloadJs from 'downloadjs';
+import { Typography, FormGroup } from '@material-ui/core';
+import { Pagination } from '@material-ui/lab';
+import { Button, LoadingContainer, Checkbox as BaseCheckbox } from '@tupaia/ui-components';
+import { useEntity, useProject } from '../../../api/queries';
+import { useExportDashboard } from '../../../api/mutations';
+import { PDFExport } from '../../../views';
+
+const ButtonGroup = styled.div`
+  padding-top: 2.5rem;
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+`;
+
+const PrimaryContext = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  width 100%;
+  align-items: start;
+`;
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  flex-basis: 83.3333%;
+`;
+
+const ExportSettingsContainer = styled.div`
+  height: 100%;
+  width: 60%;
+  display: flex;
+  flex-direction: column;
+  margin-right: 2rem;
+`;
+
+const ExportSetting = styled.div`
+  border: ${({ theme }) => theme.palette.text.secondary};
+  border-width: 0.1rem;
+  border-style: solid;
+  border-radius: 10px;
+
+  .MuiFormGroup-root {
+    align-content: start;
+  }
+
+  .MuiFormControlLabel-label {
+    font-size: 0.875rem;
+  }
+
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  padding: 1rem;
+`;
+
+const Legend = styled.legend`
+  color: ${({ theme }) => theme.palette.text.primary};
+  font-size: 1rem;
+  font-weight: ${({ theme }) => theme.typography.fontWeightMedium};
+`;
+
+const Checkbox = styled(BaseCheckbox)`
+  margin: 0.5rem 0 0 1rem;
+  .MuiButtonBase-root {
+    padding: 0;
+    margin-right: 0.5rem;
+  }
+  label {
+    padding: 0.5rem 0 0 0.5rem;
+  }
+`;
+
+const PreviewPanelContainer = styled.div`
+  height: 100%;
+  width: 38%;
+  display: flex;
+  flex-direction: column;
+`;
+
+const ExportSettingsInstructionsContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+  padding-bottom: 0.7rem;
+`;
+
+const PreviewHeaderContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 0.3rem;
+`;
+
+const PreviewPagination = styled(Pagination)`
+  .MuiPaginationItem-page {
+    font-size: 0.7rem;
+  }
+`;
+
+const PreviewContainer = styled.div`
+  display: flex;
+  background-color: white;
+  height: 30rem;
+  min-width: 20rem;
+  overflow: auto;
+`;
+
+const Instructions = styled(Typography)`
+  color: ${({ theme }) => theme.palette.text.primary};
```

Comment:
This isn't necessary to actually change, but if you ever want to easily change the colour of the text to a particular variable, you can set an attribute on the `Typography` element of `color="textPrimary"`. Usually it will default to be primary colour anyway though